src/utils/applyMultiplier.js

```diff
@@ -1,6 +1,8 @@
-function applyFactor(factor) {
-  return (value) => {
-    if (value === undefined || value === null) {
+import makeResponsiveTransform from './makeResponsiveTransform'
+
+export default function applyMultiplier(factor = 1) {
+  return makeResponsiveTransform((value) => {
+    if (value == null) {
```

Comment:
In general, why did you change these helpers?
They're still supposed to do the same, right?

Reply:
`value == null` is the same as `value === undefined || value === null`.
As for why I changed this helpers, it's because initially, I had a `grid-template-areas` implementation where a matrix was passed and I needed to transform it to string considering responsive values. So I've created `makeResponsiveTransform` function.
Probably, I should have reverted everything back, after I stopped using `makeResponsiveTransform` for `grid-template-areas`, but I thought that it might be useful in the future if we want to transform values some other way.